app/assets/stylesheets/utilities/_space-misc.scss

```diff
@@ -3,44 +3,16 @@
 .mxn-tiny { margin-left: -$space-tiny; margin-right: -$space-tiny; }
 .mb-tiny { margin-bottom: $space-tiny; }
 .ml-tiny { margin-left: $space-tiny; }
-.mr-tiny { margin-right: $space-tiny; }
 .mt-tiny { margin-top: $space-tiny; }
-.pb-tiny { padding-bottom: $space-tiny; }
 .pt-tiny { padding-top: $space-tiny; }
-.px-tiny { padding-left: $space-tiny; padding-right: $space-tiny; }
 .py-tiny { padding-bottom: $space-tiny; padding-top: $space-tiny; }
 
-.mb-12p { margin-bottom: 12px; }
-.mt-12p { margin-top: 12px; }
 .px-12p { padding-left: 12px; padding-right: 12px; }
 .py-12p { padding-bottom: 12px; padding-top: 12px; }
 
 .pl-24p { padding-left: 24px; }
 
-.mb-40p { margin-bottom: 40px; }
-
-.mtn1 { margin-top: -$space-1; }
-.mtn2 { margin-top: -$space-2; }
-.mtn3 { margin-top: -$space-3; }
-
 @media #{$breakpoint-sm} {
   .sm-mr-20p { margin-right: 20px; }
   .sm-ml-28p { margin-left: 28px; }
-  .sm-mtn2 { margin-top: -$space-2; }
-  .sm-mtn3 { margin-top: -$space-3; }
-}
-
-.margin-top-neg-4 {
```

Comment:
Some of these design-system extensions are still in use, but they've been made available directly through newer releases of USWDS.

https://github.com/uswds/uswds/releases/tag/v2.12.0

>Added more negative margin tokens to margin utilities. Now you can use utilities like `margin-neg-4` and `margin-neg-5`. (https://github.com/uswds/uswds/pull/4212)

See source: https://unpkg.com/uswds@2.12.1/dist/css/uswds.css

We inherit USWDS 2.12.1 as of https://github.com/18F/identity-idp/pull/5383.